applications/nrf_desktop/doc/smp.rst

```diff
@@ -13,6 +13,7 @@ The |smp| is responsible for performing a Device Firmware Upgrade (DFU) over Blu
 You can perform the DFU using for example the `nRF Connect for Mobile`_ application.
 The :guilabel:`DFU` button appears in the tab with the connected Bluetooth® devices.
 After pressing the button, you can select the :file:`*.bin` file that is to be used for the firmware update.
+See the :ref:`nrf_desktop_image_transfer_over_smp` section for information about which binary file should be used for the DFU image transfer.
```

Comment:
Shouldn't we add this sentence to the dfu_mcumgr.rst file?

Reply:
Will do under task that applies remaining review comments to the file